### PR TITLE
Downgrade to yarn 0.28.4

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

downgrades yarn

## What is the value of this and can you measure success?

maybe fix the seg fault ¯\_(ツ)_/¯

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "engines": {
-    "yarn": "1.0.2"
+    "yarn": "0.28.4"
```
Comment (TBonnin) [inline package.json]: big jump 😮

**Review by TBonnin — COMMENTED** [review]
not sure this is the source of the problem. Would it cause any problem to downgrade back to `0.28`?

**Comment by sndrs** [comment]
we did the upgrade last week, no seg faults before then. the lock files are the same so shouldn't be

**Review by SiAdcock — APPROVED** [review]
👍

**Comment by SiAdcock** [comment]
If we downgrade and continue to see seg faults, we can at least rule yarn out 😅

**Comment by sndrs** [comment]
i won't merge this now, but feel free tomorrow/monday

**Pull request merged**